app/Http/Controllers/Frontend/SimBriefController.php

```diff
@@ -50,7 +50,7 @@ public function generate(Request $request)
         $flight_id = $request->input('flight_id');
         $aircraft_id = $request->input('aircraft_id');
         $flight = $this->flightRepo->with(['subfleets'])->find($flight_id);
-        $flight = $this->fareSvc->getReconciledFaresForFlight($flight);
+        // $flight = $this->fareSvc->getReconciledFaresForFlight($flight);
```

Comment:
Why's this commented out?

Reply:
We do not need it anymore, it was needed when we did not had the aircraft selection and info in hand. Now with the aircraft model we can get the subfleet and fare information. You can see how i get the fare and capacities in the form ($aircraft->subfleet->fare as $fare)